### ISSUE TITLE
How to play my favorites continuously?

### ISSUE THREAD

**Issue by soulawaker** [r0man/soundklaus.el#27]
Can soundklaus play my-favorite continuously with or without shuffle?

**Comment by r0man** [comment]
@soulawaker No, not at the moment. PR welcome. Playing music continuously and shuffling could be possible via EMMS I think, but not sure.
